DigitalLearningSolutions.Data.Tests/Services/CourseTopicsServiceTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿namespace DigitalLearningSolutions.Data.Tests.Services
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using DigitalLearningSolutions.Data.DataServices;
+    using DigitalLearningSolutions.Data.Models.Common;
+    using DigitalLearningSolutions.Data.Services;
+    using FakeItEasy;
+    using FluentAssertions;
+    using NUnit.Framework;
+
+    public class CourseTopicsServiceTests
+    {
+        private ICourseTopicsDataService courseTopicsDataService = null!;
+        private CourseTopicsService courseTopicsService = null!;
+
+        [SetUp]
+        public void Setup()
+        {
+            courseTopicsDataService = A.Fake<ICourseTopicsDataService>();
+            courseTopicsService = new CourseTopicsService(
+                courseTopicsDataService
+            );
+        }
+
+        [Test]
+        public void GetActiveTopicsAvailableAtCentre_calls_data_service()

```

Comment:
This test name describes one aim, but the test itself is asserting on the `call` and also what `is returned`.
To match the convention could I suggest the test name change slightly to match the expectations:
`GetActiveTopicsAvailableAtCentre_calls_data_service*_and_returns_expected_topics*()`